backend/gtime/gtime.go

```diff
@@ -85,3 +85,35 @@ func parse(inp string) (time.Duration, string, error) {
 
 	return time.Duration(num), string(result[2]), nil
 }
+
+// FormatInterval converts a duration into the kbn format e.g. 1m 2h or 3d
```

Comment:
Nit. kbn is legacy, can we write "into the units that Grafana uses" or something to be inline with other functions in this package?